src/System.Net.Http.WinHttpHandler/src/Resources/Strings.resx

```diff
@@ -121,4 +121,7 @@
   <data name="net_http_content_stream_already_read" xml:space="preserve">
     <value>The stream was already consumed. It cannot be read again.</value>
   </data>
+  <data name="PlatformNotSupported_WinHttpHandler" xml:space="preserve">
+    <value>WinHttpHandler is designed to be used only on .NET Framework or .NET Core platforms.  It is not designed to be used for Windows Store Applications (UWP)  nor Linux platforms.</value>
```

Comment:
>Do we not consider UWP to be a .NET Core platform?

I don't think we represent it as a .NEt Core platform. I see that as an implementation detail. It existed before .NEt Core, of course.